bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/igrins2/blueprint/SpIgrins2Blueprint.java

```diff
@@ -25,15 +25,15 @@ public NoddingOption getNoddingOption() {
     }
 
     public SpIgrins2Blueprint(ParamSet paramSet) {
-        noddingOption = Pio.getEnumValue(paramSet, PARAM_NODDING, NoddingOption.KEEP_TARGET_IN_SLIT);
+        noddingOption = Pio.getEnumValue(paramSet, PARAM_NODDING, NoddingOption.NOD_ALONG_SLIT);
```

Comment:
The blueprints are stored in the science program, so I think we may need to migrate the nodding option unfortunately assuming this was in the 2024A model?

Reply:
That wasn't very clear to me. We haven't imported any IG2 p1 proposal so far

Reply:
Right, I guess this is new for this semester and we don't use `NoddingOption` in the Phase 2 model.  It should be fine. hehe